gitbutler-ui/src/lib/components/TextBox.svelte

```diff
@@ -60,20 +72,54 @@
 			{placeholder}
 			{spellcheck}
 			{disabled}
+			min={minVal}
+			max={maxVal}
 			{...type == 'password' && showPassword ? { type: 'text' } : { type }}
 			class="text-input textbox__input text-base-13"
 			class:textbox__readonly={type != 'select' && readonly}
 			class:select-none={noselect}
 			class:select-all={selectall}
+			style:text-align={textAlign}
 			bind:value
 			bind:this={htmlInput}
 			on:click
 			on:mousedown
-			on:input={(e) => dispatch('input', e.currentTarget.value)}
+			on:input={(e) => {
+				dispatch('input', e.currentTarget.value);
+
+				isInputValid = e.currentTarget.checkValidity();
+			}}
 			on:change={(e) => dispatch('change', e.currentTarget.value)}
 			on:keydown={(e) => dispatch('keydown', e)}
 		/>
 
+		{#if type == 'number'}
+			<div class="textbox__count-actions">
+				<button
+					class="textbox__count-btn"
+					on:click={() => {
```

Comment:
From quick testing, it seems like the setting is only being when typing numbers, but not when just clicking the +/-

Reply:
So maybe these buttons should also dispatch `change` events?

Reply:
I went ahead and made this change, so please let me know if you disagree

Reply:
👍